src/Common/tests/System/Collections/DictionaryExtensions.cs

```diff
@@ -1,20 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-namespace System.Collections.Generic
-{
-    internal static class DictionaryExtensions
```

Comment:
Actually we would need to add the Remove API that is being added to `CollectionExtensions` in order for all the tests to pass in Desktop right, @stephentoub ?